agent_pool.go

```diff
@@ -172,6 +174,12 @@ type AgentPoolUpdateOptions struct {
 
 	// A new name to identify the agent pool.
 	Name *string `jsonapi:"attr,name"`
+
+	// True if the agent pool is organization scoped, false otherwise.
+	OrganizationScoped bool `jsonapi:"attr,organization-scoped"`
```

Comment:
Since this is (presumably) not a required attribute, you probably want to use a pointer here.